<launch>
  <node pkg="opt_utils" type="roi_viewer" name="roi_viewer" output="screen">
  <!-- mode = {roi_display} -->
  <param name="mode" type="string" value="roi_display"/> -->
  <!-- Note: label should be updated to list of values -->
  <param name="label" type="int" value="1" />
  <remap from ="input_image" to="/camera/rgb/image_raw"/>
  <remap from ="input_rois" to="/ground_based_people_detector/detections"/>  
  </node>
</launch>

